orderer/consensus/hlmirbft/node.go

```diff
@@ -142,3 +153,51 @@ func (n *node) lastIndex() uint64 {
 	return i*/
 	return 0
 }
+
+
+
+//JIRA FLY2- 66 Proposed changes:Implemented the PersistSnapshot functionality to persist the snaps to local files
+func (n *node) PersistSnapshot(seqNo uint64, Data []byte) error {
+	if err := os.MkdirAll(n.chain.opts.SnapDir, os.ModePerm); err != nil {
+		return errors.Errorf("failed to mkdir '%s' for snapshot: %s", n.chain.opts.SnapDir, err)
+	}
+
+	TimeStamp := time.Now().Unix()
```

Comment:
@harrymknight I'd appreciate your thoughts on this. I think marking snapshots with a timestamp is brave at best due to the chances of then resyncing another node from a snapshot. So timestamp is not going to work as an identifier -- it needs to be block height.

Reply:
@renjithpta Raise a ticket for this change and we will put it in the next sprint.

Reply:
@needhamj JIRA ticket ?